### DISCUSSION TITLE
Lazy Loading Components

### DISCUSSION THREAD

**Discussion by JHerwig** [nuxt/nuxt#8769] [Questions (Nuxt 2)]
I would like to load all layout components in the same time with skelletons. These skelletons should be replaced if i got results from my api. Currently i am using the `fetch` hook to perform my axios calls.
Sadly my sidebar is blocking somehow and is only shown if all data are loaded.

**Question**: How to realize Lazy Loading correct?

**Comment by regenrek** [comment]
Hi, did you read https://github.com/nuxt/components#lazy-imports

Regards
